totalRP3/Core/EncodingUtil.lua

```diff
@@ -0,0 +1,177 @@
+-- Copyright The Total RP 3 Authors
+-- SPDX-License-Identifier: Apache-2.0
+
+TRP3_EncodingUtil = {};
+
+--
+-- PEM Encoding and Decoding Utilities
+--
+
+function TRP3_EncodingUtil.IsPEMEncodingSupported()
+	return C_EncodingUtil and C_EncodingUtil.EncodeBase64;
+end
```

Comment:
Temp function that'll be cleaned up when Classic Era gets its shit together. 

Currently this is used to branch into using the old export format, rather than trying to shoehorn together Lua-based implementations of all these things.